### ISSUE TITLE
[coding-standards] Please upgrade dependency to "symplify/easy-coding-standard": "^7.3.18"

### ISSUE THREAD

**Issue by spagr** [shopsys/shopsys#1859]
Hello!
I would like to test your ForbiddenDumpSniff and ForbiddenDumpFixer ECS rules/sevices, but I work in Symfony 5, in which ECS only works well in versions ^ 7.3.18.  

Could you please upgrade the dependency of this package?  

I cloned repo, updated the dependencies, and ran the tests.
No errors occurred :  
![image](https://user-images.githubusercontent.com/6697111/82349062-eff5ef00-99f9-11ea-9d0f-be388b6114ae.png)

Attaching a modified composer.json :  
```json
{
    "name": "shopsys/coding-standards",
    "type": "library",
    "description": "Coding standards definition compatible with PSR-2",
    "keywords": ["code style", "coding standards", "PSR-2", "code quality"],
    "license": "MIT",
    "authors": [
        {
            "name": "Shopsys",
            "homepage": "https://www.shopsys.com/"
        }
    ],
    "require": {
        "php": "^7.2",
        "ext-tokenizer": "*",
        "friendsofphp/php-cs-fixer": "^2.14.0",
        "jakub-onderka/php-parallel-lint": "^0.9",
        "object-calisthenics/phpcs-calisthenics-rules": "^3.1",
        "slevomat/coding-standard": "^6.3.5",
        "squizlabs/php_codesniffer": "^3.5.0",
        "symplify/package-builder": "^7.3.18",
        "symplify/easy-coding-standard": "^7.3.18",
        "symfony/finder": "^4.4"
    },
    "require-dev": {
        "phpunit/phpunit": "^8.0",
        "symplify/easy-coding-standard-tester": "^7.3.18"
    },
    "autoload": {
        "psr-4": {
            "Shopsys\\CodingStandards\\": "src/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\CodingStandards\\": "tests/"
        }
    },
    "extra": {
        "branch-alias": {
            "dev-master": "9.0.x-dev"
        }
    }
} 
```

I'm not sure how and where to properly create a monorepo pull request, so I solve it this special way ...

Thank you in advance

Martin

**Comment by pesektomas** [comment]
Hello @spagr,

you can create moorepo pull request very easy :) 

First, fork this repository:

```sh
git clone <your/shopsys/shopsys/fork> shopsys
cd shopsys
```

In shopsys project, please change `composer.json` and change `composer.json` in `packages/coding-standards` folder.


When I tried to update symplify packages at the last time, I had a problem with the `%vendor_dir%` variable in easy-coding-standards.yml not being overwritten. It was in version 7.2.3. If something was changed I don't known about it. But if your PR will pass, it will be great to accept your PR :)

Thank you very much for your report.

**Comment by spagr** [comment]
I tried, but no big deal ...  https://travis-ci.org/github/shopsys/shopsys/builds/689386791#L3507

**Comment by grossmannmartin** [comment]
Hi, thank you for your time :)
this looks like Travis failed on some unrelated problem. But it's strange, the current master passes.

**Comment by spagr** [comment]
It was about building patch bundle.php file, I made and committed new diff file.
I also made little update in packages/coding-standards/easy-coding-standard.yml.
Build, Lint and ECS tests were successful then.

But there is another BC issue in utils/releaser with new symplify/monorepo-builder, which appears at PhpStan stage : 
https://travis-ci.org/github/shopsys/shopsys/builds/689582440#L5069

Too bad I thought I was really close ... :wink:

**Comment by grossmannmartin** [comment]
Really appreciate your contribution :)
Hmm... bc break in a minor version of Symplify package... wonder how this could happen.

**Comment by spagr** [comment]
Even @TomasVotruba  is only human :smile:  .
Is the "utils/releaser" (and dependent packages) so covered in tests that I, who does not know the whole context and functions of this utility, can refactor it, test it and there will be a chance of still a fully functional utility?  

I can spend some time on this over the weekend.  
I'm happy to do this as part of contributing to open-source. :wink:  

So if it will not be a wasted time, I mean this won't/can't be merged into a master ...

**Comment by pesektomas** [comment]
Hello, it certainly won't be a lost time. Unfortunately, the releaser is not much (read at all) covered by the tests. It is checked only by ECS and phpstan, anyway thank you for your time, we can test the releaser manually :smile:

**Comment by spagr** [comment]
It will probably be a rainy weekend, I'll give it a chance :wink:

**Comment by spagr** [comment]
I pushed last commit few minutes ago, my local tests pass well yesterday.  
If this job https://travis-ci.org/github/shopsys/shopsys/builds/690829736 will be success, can You manually test if monorepo builder works.  

It shouldn't be a problem, in the new Symplify release only a simple message class was missing and the json tool class is extracted in a separate package, no new magic added ;-)

**Comment by spagr** [comment]
Hello @pesektomas,
is there any change to merge relating PR ?

Thank you

**Comment by TomasLudvik** [comment]
Hello, I think we will move mentioned PR to done in next weeks.

**Comment by TomasLudvik** [comment]
This has been solved in #1862, closing
